x/wasm/internal/keeper/relay_test.go

```diff
@@ -32,7 +32,7 @@ func TestOnOpenChannel(t *testing.T) {
 		},
 		"consume max gas": {
 			contractAddr: example.Contract,
-			contractGas:  MaxGas,
+			contractGas:  math.MaxUint64 / DefaultGasMultiplier,
```

Comment:
```suggestion
			contractGas:  math.MaxUint64 / types.DefaultGasMultiplier,
```

and

`math` should be also imported.